src/locales/en.js

```diff
@@ -290,7 +290,7 @@ export default {
     "saintBernardineOfSienaPriest": "Saint Bernardine of Siena, Priest",
     "saintBernardOfClairvauxAbbotAndDoctorOfTheChurch": "Saint Bernard of Clairvaux, Abbot and Doctor of the Church",
     "saintBeunoAbbot": "Saint Beuno, Abbot",
-    "saintBlaseBishopAndMartyrSaintAnsgarBishop": "Saint Blase, Bishop Martyr and Saint Ansgar, Bishop",
+    "saintBlaseBishopAndMartyrSaintAnsgarBishop": "Saint Blase, Bishop Martyr or Saint Ansgar, Bishop",
```

Comment:
You are changing the `and` to `or` in the localised title, but not in the key?
Also if it should be an `or` instead of an `and`, we should have 2 different keys (and 2 different celebration date) instead.

Reply:
You are right, I forgot to rename the key.

> Also if it should be an `or` instead of an `and`, we should have 2 different keys (and 2 different celebration date) instead.

I know how to define two keys, but how do I define a celebration (in the calendar files) with two keys ATM? I know we want to implement it, but is it already implemented? If it is not yet implemented, we should have one key (as it is right now, but replace `or` with `and`) until it is implemented.

Reply:
Yes it's now implemented. You now can simply create 2 celebrations with the same date, in the same calendar. And romcal will output all the suitable celebrations for the corresponding date (including all optional memories if it's appropriated).

Reply:
Okay. And is it implemented in the `_F-TypeScript` branch too?

Reply:
Yes.